### PR TITLE
stat/distmv: Add KL divergence for two Dirichlet distributions

### PR DESCRIPTION
**Pull request by btracey**
Please take a look.

### CODE REVIEW

File: stat/distmv/statdist.go [hunk 1]
```diff
@@ -7,6 +7,8 @@ package distmv
 import (
 	"math"
 
+	"gonum.org/v1/gonum/mathext"
+
```
Comment (kortschak) [inline stat/distmv/statdist.go]: Delete blank line

Comment (btracey) [reply stat/distmv/statdist.go]: Done.

**Comment by btracey** [comment]
Based on the discussion in #509 , I fixed all the spelling mistakes in KL and changed distance to divergence.

**Comment by btracey** [comment]
Can I get your approval here too @vladimir-ch ?

**Pull request merged**